pom.xml

```diff
@@ -5,65 +5,92 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.jitsi</groupId>
-    <artifactId>jitsi-sctp</artifactId>
+    <artifactId>sctp</artifactId>
```

Comment:
sctp4j?

Reply:
Me likey

Reply:
The old name of the archive was "sctp", but since the only thing that depends on it as far as I know is jitsi-videobridge, it's not a big deal to change it, and it's probably a good idea.

I might use jitsi-sctp rather than sctp4j tho.